### PR TITLE
HEEDLS-796 : Update "All Delegates - Remove course" with correct wireframe

### PR DESCRIPTION
**Pull request by GeorgeBlackwell**
### JIRA link
_[HEEDLS-796](https://softwiretech.atlassian.net/browse/HEEDLS-796)_

### Description
_Update "All Delegates - Remove course" with correct wording and layout._

### Screenshots
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [X] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [X] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [X] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml [hunk 1]
```diff
@@ -14,8 +14,9 @@
     @if (errorHasOccurred) {
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
     }
-    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove delegate enrolment from @Model.CourseName</h1>
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove enrolment</h1>
     <h2 class="nhsuk-heading-l word-break">Delegate: @Model.Name</h2>
+    <h2 class="nhsuk-heading-l word-break">Activity: @Model.CourseName</h2>
```
Comment (livzorn) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml:19]: Comparing the screenshot to other remove confirmation pages on the site, it looks like there's usually a little more margin under the title, which would be good to have here for consistency. I think it often comes from putting the h1 in it's own row - see RemoveGroupCourse.cshtml and RemoveGroupDelegate.cshtml for examples

File: DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml [hunk 2]
```diff
@@ -14,8 +14,9 @@
     @if (errorHasOccurred) {
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
     }
-    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove delegate enrolment from @Model.CourseName</h1>
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove enrolment</h1>
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml:17]: Think this page is listed in the automated accessibility tests, and will need that test parameter changing to match this new h1.

**Review by DanBloxham-sw — CHANGES_REQUESTED** [review]
Looks good, I agree with Olivia's comment, and I've left one more about the UI tests.

**Review by DanBloxham-sw — APPROVED** [review]
Looks good

**Pull request merged**